packages/service-library/src/web-api/api-contracts/scan-run-request.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 export interface ScanRunRequest {
     url: string;
-
+    replyUrl?: string;
```

Comment:
I think WCP team sends {url, priority} and they will send  {url, priority, replyUrl}, plz verify that the sequence and I am happy to know whether sequence matters or not.

Reply:
I don't think the order matters, cause we are parsing them as JSON object.